src/style/aframe.css

```diff
@@ -54,7 +54,7 @@
   right: 0 !important;
   bottom: 0 !important;
   z-index: 999999 !important;
-  position: fixed;
+  position: fixed !important;
```

Comment:
can you add a comment? I hope nobody needs to ever override any of these important values. perhaps these styles would be better set with a `a.fullscreen.forced` class? I could be wrong. I haven't touched this code in a while.